api/client/client.go

```diff
@@ -101,6 +104,11 @@ func NewClient(target string, optsOverrides ...grpc.DialOption) (*Client, error)
 	return client, nil
 }
 
+// Target returns the client target address.
+func (c *Client) Target() string {
+	return c.conn.Target()
+}
+
```

Comment:
Exposing this so we can build a complete report about the server the client is talking to.